table/plain_table_factory.h

```diff
@@ -204,8 +204,7 @@ static std::unordered_map<std::string, OptionTypeInfo> plain_table_type_info = {
      {offsetof(struct PlainTableOptions, full_scan_mode), OptionType::kBoolean,
       OptionVerificationType::kNormal, false, 0}},
     {"store_index_in_file",
-     {offsetof(struct PlainTableOptions, store_index_in_file),
-      OptionType::kBoolean, OptionVerificationType::kNormal, false, 0}}};
+     {0, OptionType::kBoolean, OptionVerificationType::kDeprecated, false, 0}}};
```

Comment:
Can we remove this "store_index_in_file"?

Reply:
We don't usually remove option from option file. Only tag it as kDeprecated. Otherwise, loading option files from older releases will fail, and it will be an operational overhead for people who upgrade.